cs3api4lab/api/cs3apismanager.py

```diff
@@ -493,3 +508,37 @@ def list_checkpoints(self, path):
 
     def delete_checkpoint(self, checkpoint_id, path):
         pass
+
+    @asyncify
+    def create_conflict_file(self, path):
```

Comment:
I'm not a big fan of calling this "conflict file". Those are files that were a cause of a concurrency issue and should not happen. This is more a "copy" or a "clone". And I would have that copy filename have the original filename + the user that created. Nothing more (like timestamps).

By separating these 2 cases, we also make it easier to distinguish if we want to diff in the future (although, we might want to merge in both cases).